lib/mdx.js

```diff
@@ -65,8 +65,13 @@ export async function getFilesFrontMatter(type, limit = Infinity) {
     ];
   }, []);
 
-  return parsedFiles.sort(
+  return parsedFiles
+    .filter(
+      (post) => new Date().getTime() > new Date(post.publishedAt).getTime()
```

Comment:
hey @jelitter can we compare the date part only without comparing the time to be more accurate? Maybe adding a variable `const currentDate = new Date(new Date().toDateString());` out of return

```suggestion
      (post) => currentDate.getTime() > new Date(post.publishedAt).getTime()
```

Reply:
@lavaldi I think I did it, not sure as I have never done this before, apologies if something is wrong ^^